test/Microsoft.ML.StaticPipelineTesting/Training.cs

```diff
@@ -162,7 +165,7 @@ public void SdcaBinaryClassificationNoClaibration()
 
             var data = model.Read(dataSource);
 
-            var metrics = BinaryClassifierEvaluator.Evaluate(data, r => r.label, r => r.preds);
```

Comment:
>r => r.label, [](start = 67, length = 13)

this hasn't changed, but without an example, it's hard to figure out what to put in here.. there's no way to have a string, rather than the function?

Reply:
There is, if you do `data.AsDynamic`

---
In reply to: [219330956](https://github.com/dotnet/machinelearning/pull/967#discussion_r219330956) [](ancestors = 219330956)